joomla/joomla-cms
# [4.0] Cassiopeia remove mixin - margin

This mixin was added #27861

But then the only place it was used was removed during #31520

To test

- make a copy of templates\cassiopeia\css\template.css
- apply pr
- regenerate the css (node build/build.js --compile-css)
- compare the copy and the new template.css
- There should be no differences.
